types/enzyme-adapter-react-16/index.d.ts

```diff
@@ -1,15 +1,15 @@
-// Type definitions for enzyme-adapter-react-16 1.0
+// Type definitions for enzyme-adapter-react-16 1.1
```

Comment:
The version number here is informative only and should match one from the original library: https://www.npmjs.com/package/enzyme-adapter-react-16.
You can leave 1.0 or use the latest version number of enzyme-adapter-react-16 (1.6.0) since the API did not change.